include/veriblock/pop/blockchain/alt_block_tree.hpp

```diff
@@ -115,19 +114,16 @@ extern template struct BaseBlockTree<AltBlock>;
 //!         // here, we assume that candidate has all txes downloaded and block is fully available
 //!
 //!         // compare current tip to a candidate
-//!         int result = tree.comparePopScore(tip->getHash(), candidate->getHash());
+//!         int result = tree.activateBestChain(tip->getHash(), candidate->getHash());
 //! ```
 //!
-//! @note after AltBlockTree::comparePopScore AltBlockTree always corresponds to a state, as if winner chain have been applied.
+//! @note after AltBlockTree::activateBestChain AltBlockTree always corresponds to a state, as if winner chain have been applied.
 //!
 //!```cpp
 //!         if(result < 0) {
 //!             // candidate has better POP score.
 //!             // tree already switched to candidate chain.
 //!
-//!             UpdateTip(candidate->getHash());
-//!             // NOTE: update `tip`, otherwise old tip will be passed to first arg,
-//!             // and comparePopScore will die on assert
```

Comment:
Why did you delete this?
Users must update tip of their block tree, otherwise ALT tree in alt-cpp and block tree in altchain won't be in sync

Reply:
I thought it is related to overrideTip call which is obsolete after this refactoring.